### ISSUE TITLE
Calculate weights for pallet dispatchables

### ISSUE THREAD

**Issue by vgeddes** [Snowfork/polkadot-ethereum#161]
We'll need to perform benchmarking in order to calculate the appropriate weights.

**Comment by philipstanislaus** [comment]
Benchmarks have been added with weights in a couple of PRs, e. g. #307. The benchmark have run on a personal machine so far (see https://github.com/Snowfork/polkadot-ethereum/pull/307#issuecomment-809885585), but should be replicated in a dedicated machine, ideally with resources similar to the ones used by the benchmarks done by Parity for Polkadot/Substrate runtimes, see https://www.shawntabrizi.com/substrate-graph-benchmarks/docs/#/assumptions?id=hardware for reference.

**Comment by Rizziepit** [comment]
After evaluating our options on AWS, we decided to go with a `c5a.2xlarge` instance type as our baseline (see discussion at https://github.com/Snowfork/polkadot-ethereum/pull/388). I recalculated weights in https://github.com/Snowfork/polkadot-ethereum/pull/395 and updated the baseline specs in our docs in https://github.com/Snowfork/polkadot-ethereum/pull/396. This wraps up benchmarking (or shall I say v1 benchmarking?).

Some important notes:
* The weights were calculated on a shared host. To guarantee perf, we might want to run our nodes on dedicated hosts (cc @vgeddes).
* We should regularly recalculate these weights post-launch.
* Whenever a pallet changes, we must ensure the benchmarks still represent the worst-case performance.
